File: plugins/claude-md-plugin/core/tests/fixtures/go/export_candidates.go
package auth

import "time"

const MaxRetries int = 3
const DefaultTimeout time.Duration = 30 * time.Second

// unexported const - should NOT be captured
const internalLimit int = 100

type UserID = string
type TokenResult = map[string]interface{}

// unexported type alias - should NOT be captured
type internalType = int

type Config struct {
	Timeout int
}

func ProcessItem(item string) bool {
	return len(item) > 0
}

File: plugins/claude-md-plugin/core/tests/fixtures/python/export_candidates.py
# Python export candidates fixture
# Tests: UPPER_CASE constants, type aliases

from typing import Union, Optional, List

MAX_RETRIES = 3
DEFAULT_TIMEOUT = 30
API_BASE_URL = "https://api.example.com"

UserId = Union[str, int]
TokenResult = Optional[dict]
ItemList = List[str]

# Should NOT be captured (not UPPER_CASE, not PascalCase type alias)
_internal_counter = 0
simple_var = "hello"


class UserConfig:
    pass


def process_item(item: str) -> bool:
    return len(item) > 0
